src/DetailsView/reports/components/report-sections/outcome-summary-bar.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { kebabCase } from 'lodash';
+import { CheckIcon } from '../../../../common/icons/check-icon';
+import { CircleIcon } from '../../../../common/icons/circle-icon';
+import { CrossIcon } from '../../../../common/icons/cross-icon';
+import { NamedSFC } from '../../../../common/react/named-sfc';
+import { SectionProps } from './report-section-factory';
+
+const iconMap = {
+    pass: <CheckIcon />,
+    'not applicable': <CircleIcon />,
+    fail: <CrossIcon />,
+};
+
+const outcomeText = {
+    pass: 'Passed',
+    'not applicable': 'Not applicable',
+    fail: 'Failed',
+};
+
+type OutcomeType = 'pass' | 'fail' | 'not applicable';
```

Comment:
Can we use the type  ``OutcomeType`` from ``outcome-type.ts`` ?

Reply:
After a second look, the 'outcome' here is a different concept than the one in assessment. In that case, should we use another name to avoid confusion?

Reply:
yeap... I though about refactoring the original component, make it generic (literally <TOutcomeType>)... but it has some hard dependencies so I decide to take a different approach for now. This is something that goes to the clean up & refactoring list though.

Reply:
Sounds good. let me make a note in that story so that we don't forget.